OrbitGl/CallStackDataView.h

```diff
@@ -37,7 +37,7 @@ class CallStackDataView : public DataView {
  protected:
   void DoFilter() override;
 
-  std::unique_ptr<CallStack> callstack_;
+  const CallStack* callstack_;
```

Comment:
Would it be a lot of effort to change this into a shared_ptr? I think we should avoid working with raw pointers if possible.

Reply:
We should probably discuss this. We want to avoid shread_ptr because they mask ownership and they come with the cost of thread-safe reference counting. The long version of modivation is here https://groups.google.com/a/google.com/g/stadia-profiling-dev-muc/c/MFBcOfQQQ80/m/JME_UsJyAQAJ

Reply:
I didn't get it: why do you think we need a shared_ptr here? I don't think that we need to share ownership here...

Reply:
Maybe just use a copy, or an optional? and avoid pointers at all?

Reply:
Thanks for pointing out the mailthread, Dimitry. Maybe we should discuss this offline - I have a different opinion on the use of shared / weak ptr, but I'm happy to discuss and change my mind. Let's touch it in the daily?

Reply:
Agree, let's discuss it in daily today

Reply:
This is still only one callstack, right? So we would not loose much if we would just copy it here and don't have a pointer at all.